src/Compilers/CSharp/Test/Symbol/Compilation/SymbolSearchTests.cs

```diff
@@ -24,6 +24,12 @@ public void TestSymbolFilterNone()
                 var compilation = GetTestCompilation();
                 compilation.GetSymbolsWithName(n => true, SymbolFilter.None);
             });
+
+            Assert.Throws<ArgumentException>(() =>
+            {
+                var compilation = GetTestCompilation();
+                compilation.GetSymbolsWithName("", SymbolFilter.None);
```

Comment:
>GetSymbolsWithName [](start = 28, length = 18)

I expect a similar case for `ContainsSymbolsWithName` (passing in a name, but `SymbolFilter.None`)

Reply:
will do.

Reply:
Done.  #resolved